.github/workflows/ci-docs-tests.yml

```
@@ -36,7 +36,7 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Set up Python ${{ env.DEFAULT_PYTHON_VERSION }} 🐍
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@dc73133d4da04e56a135ae2246682783cc7c7cb6  # v.2.2.2
```

```suggestion
      uses: actions/setup-python@dc73133d4da04e56a135ae2246682783cc7c7cb6  # v2.2.2
```

The same thing in the rest of places

---

🕺 Fixed